package.json

```diff
@@ -1,137 +1,101 @@
+    "@typescript-eslint/parser": "5.59.5",
     "autoprefixer": "10.4.7",
-    "axios": "^0.21.1",
-    "benchmark": "^2.1.4",
-    "classnames": "2.2.6",
     "clean-webpack-plugin": "^0.1.19",
-    "codecov": "^3.1.0",
     "copy-webpack-plugin": "^11.0.0",
     "cspell": "6.13.3",
-    "css-loader": "^6.7.1",
-    "eslint": "8.26.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-jsdoc": "^39.6.2",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.26.1",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-webpack-plugin": "^3.1.1",
-    "fork-ts-checker-webpack-plugin": "^7.2.0",
-    "glob": "^8.0.3",
+    "css-loader": "^6.7.3",
+    "eslint": "8.42.0",
+    "eslint-plugin-jsdoc": "^46.8.2",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-webpack-plugin": "^4.0.1",
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "glob": "^10.2.7",
     "html-loader": "0.5.5",
     "identity-obj-proxy": "3.0.0",
-    "jest": "^29.3.1",
-    "jest-environment-jsdom": "^29.3.1",
-    "jest-environment-jsdom-fifteen": "^1.0.2",
-    "jscs": "^3.0.7",
-    "jsdom": "~11.3.0",
-    "jshint": "^2.9.6",
-    "jshint-stylish": "^2.1.0",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
     "lodash": "4.17.21",
-    "memoize-one": "5.1.1",
     "mini-css-extract-plugin": "2.6.1",
-    "moment": "2.24.0",
-    "node-sass": "^7.0.1",
+    "moment": "2.29.4",
     "postcss": "8.4.14",
     "postcss-loader": "7.0.1",
     "postcss-reporter": "7.0.5",
     "postcss-scss": "4.0.4",
-    "prettier": "^2.5.0",
+    "prettier": "^2.8.7",
     "prop-types": "15.7.2",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-popper": "^2.2.3",
     "react-table-6": "6.11.0",
-    "react-test-renderer": "17.0.2",
-    "react-transition-group": "4.3.0",
     "react-use": "17.4.0",
     "replace-in-file-webpack-plugin": "^1.0.6",
-    "rxjs": "6.6.3",
-    "sass": "1.56.1",
-    "sass-loader": "13.2.0",
-    "semver": "^7.3.2",
-    "style-loader": "3.3.1",
+    "sass": "1.63.2",
+    "sass-loader": "13.3.1",
+    "semver": "7.5.4",
+    "style-loader": "3.3.3",
     "swc-loader": "^0.2.3",
-    "tether-drop": "^1.4.2",
-    "ts-jest": "26.4.1",
-    "ts-loader": "4.4.1",
-    "ts-node": "^10.5.0",
-    "tsconfig-paths": "^4.1.0",
-    "tslint": "^6.1.3",
-    "typescript": "4.8.2",
-    "webpack": "^5.69.1",
-    "webpack-cli": "^4.9.2",
+    "ts-node": "^10.9.1",
+    "tsconfig-paths": "^4.2.0",
+    "typescript": "4.8.4",
+    "webpack": "^5.86.0",
+    "webpack-cli": "^5.1.4",
     "webpack-livereload-plugin": "^3.0.2",
     "webpack-remove-empty-scripts": "^1.0.1"
   },
-  "engines": {
-    "node": ">=16"
-  },
   "resolutions": {
-    "js-yaml": "^3.13.1",
-    "lodash": "~4.17.13",
-    "set-value": "^2.0.1",
-    "mixin-deep": "^1.3.2",
-    "minimatch": "^3.0.2",
-    "fstream": "^1.0.12"
-  }
+    "jackspeak": "2.1.1"
```

Comment:
Had to pin this dependency because of an issue in yarn 1. See https://github.com/storybookjs/storybook/issues/22431#issuecomment-1630086092 for more info.